benchmarks/c/miscellaneous/jumpFromOutside.c

```diff
@@ -0,0 +1,28 @@
+#include <stdint.h>
+#include <assert.h>
+#include <dat3m.h>
+
+volatile int32_t x = 0;
+
+int main()
```

Comment:
I think this benchmark should have some assertion. I also think you can make the loop you jump into explicit rather than using gotos. And then I would rename the benchmark to `jumpIntoLoop`

It could look like this
```
int i = __VERIFIER_nondet_int();
if (i == 42) goto L;

for (i = 0; i < 5; i++) {
L:
// Maybe add a memory operation
}
assert i <= 5;
```